CHANGELOG.md

```diff
@@ -1,10 +1,78 @@
 # CHANGELOG
 
+## Cardano SL 1.3.0 (Mainnet)
+
+### Features
+
+- The V1 API and its test coverage are finalized (CBR-101).
+
+- Expose SubscriptionStatus as part of /api/v1/node-info (CBR-186).
+
+- Better error message for missing charset (Wallet Backend - CBR-223).
+
+- Create AVVM mnemonic page screenshot (CBR-281).
```

Comment:
How is this a feature of Cardano? It is a screenshot created for use in the helpdesk.